.gitlab/functional_test/system_probe.yml

```
@@ -37,6 +37,20 @@
   - $S3_CP_CMD $S3_ARTIFACTS_URI/minimized-btfs-${ARCH}.tar.xz /tmp/minimized-btfs.tar.xz
   - cp /tmp/minimized-btfs.tar.xz $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files/minimized-btfs.tar.xz
 
+# This dummy job is added here because we want the functional_tests stage to start at the same time as kernel_matrix_testing stage.
+# The ebpf-platform team is trying to measure the time from the start of the pipeline to the completion of the kernel_matrix_testing and functional_tests stages, to measure improvement.
+# The CI visibility product currently does not provide a way to measure this in the datadog app.
+# We are trying to hack around this by having jobs which are triggered at the start of a pipeline.
+# For functional tests this is the 'kitchen_test_dummy_job_tmp' and for kernel_matrix_testing it is the 'pull_test_dockers*' jobs.
+# This way the stage.duration metric will allow us to measure what we want.
+# This will most likely be temporary, until we decide which approach to move forward with for testing in the future.
+kitchen_test_dummy_job_tmp:
+  stage: functional_test
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/btf-gen$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BTF_GEN_BUILDIMAGES
```

```suggestion
  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/btf-gen$DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BTF_GEN_BUILDIMAGES
```

(this is also wrong in one other place, which will need to be fixed separately)